### PR TITLE
Launch options page on first run

### PR DESCRIPTION
**Pull request by arpitnext**
This would prompt users to set their token.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
+1 for this feature!

File: extension/main.js [hunk 1]
```diff
@@ -73,6 +73,14 @@
 	chrome.alarms.create({when: Date.now() + 2000});
 	chrome.alarms.onAlarm.addListener(update);
 	chrome.runtime.onMessage.addListener(update);
+	
+	// launch options page on first run
+
+	chrome.runtime.onInstalled.addListener(function(details) {
+		if (details.reason == "install") {
+			chrome.tabs.create({url: "options.html"});
```
Comment (Narno) [inline extension/main.js]:
```javascript
chrome.runtime.openOptionsPage();
```

https://developer.chrome.com/extensions/runtime#method-openOptionsPage

Comment (arpitnext) [inline extension/main.js]: Definitely, better approach.

**Comment by sindresorhus** [comment]
Tests are failing: https://travis-ci.org/sindresorhus/github-notifier-chrome/builds/90161650

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request closed without merging**